blog/2023-09-15-teams-feature-overview/index.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Make the dream work! An overview of Stately Studio’s teams feature
+description: Getting the most out of Stately Studio’s teams feature
+tags: [stately studio, xstate, teams, machine versions, machine locking]
+authors: [gavin]
+slug: using-teams-in-stately-studio
+image: /blog/2023-09-15-stately-teams-overview.png
+date: 2023-09-19
+---
+
+Our team knew early on that users needed the ability to share machines in the Studio and build on each other’s work.
+We also needed the Studio to be an effective tool for teams to work together and share context.
+But how do we do that safely, making sure the right eyes were on the right machines?
+And how do we protect against accidental mishaps that have plagued devs since the dawn of the computer?
+Being a distributed dev team ourselves, we’ve shared these same pain points, and decided to build a solution directly in the Studio.
+Enter [Stately teams](https://stately.ai/docs/teams), our way to provide privacy and safety while allowing effortless collaboration.
+
+<!--truncate-->
+
+## Why would I need this?
+
+Great question, and we’re glad you asked!
+While we have the [Discover](https://stately.ai/registry/discover) page, which showcases over 20 thousand(!) community-created machines, we know that statecharts can contain critical details about core algorithms or can have processes that give businesses a competitive edge.
+Many users expressed interest in keeping these types of machines private, and we listened.
+
+### Sharing machine visualizations with teammates
+
+Developing software is collaborative by nature. To get any product launched, bug fixed, or feature shipped, teams need to work together under the same shared context.
+As projects grow in scope and complexity, it gets easier for team members to lose track of the bigger picture.
+Team members may get lost in the weeds of their own work while the rest of the project changes around them, or they may not have the full context of the project and make decisions that don’t align with the team’s goals.
+This is where the Studio’s teams feature comes in.
+Visualizing app logic as statecharts allows teams to see the entire project at a glance, and the ability to share these visualizations empowers each member to make well-informed decisions, regardless of discipline.
+Every teammate may not be a dev, but we’re confident that every teammate can benefit from a well thought out flowchart!
+
+## Teams 101
+
+Teams is an exclusive feature for [**Pro** and **Enterprise** users](https://stately.ai/pricing) that allows them to collaborate on different projects without needing to make them publicly visible. Team owners have the ability to assign roles that map closely with their team members’ intended set of responsibilities.
+
+![team screenshot](./teamVNext.png)
```

Comment:
```suggestion
![The Stately Studio Next team members tab. Under Owners/Admin, Kevin is labeled as Owner. David is an admin. Laura, Farzad and Gavin are listed under Editors. Anders, cirephe, and Mateusz are listed under Viewers.](./teamVNext.png)
```

Reply:
ah, i take it this also helps with assistive tech! Updating and remembering!